README.md

```diff
@@ -1,2 +1,218 @@
-# momento-go-redis-client
-Momento-backed implementation of the go-redis client
+<img src="https://docs.momentohq.com/img/logo.svg" alt="logo" width="400"/>
+
+[![project status](https://momentohq.github.io/standards-and-practices/badges/project-status-incubating.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
+[![project stability](https://momentohq.github.io/standards-and-practices/badges/project-stability-alpha.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
+
+# Momento Go-Redis Compatibility Client
+
+## What and Why?
+
+This project provides a Momento-backed implementation of [go-redis](hhttps://github.com/redis/go-redis).
+The goal is to provide a drop-in replacement for [go-redis](hhttps://github.com/redis/go-redis) so that you can
+use the same code with either a Redis server or with the Momento Cache service!
+
+You can use Momento as your cache engine for any Go project that support a redis-backed cache.
+
+## Usage
+
+To switch your existing `go-redis` application to use Momento, you only need to change the code where you construct your client object:
+
+<table>
+<tr>
+ <td width="50%">With go-redis client</td>
+ <td width="50%" valign="top">
```

Comment:
Sorry to keep harping on this, but can you just pull it out of a table completely? When I look at https://github.com/momentohq/momento-go-redis-client/tree/final-impl-go-redis-client, I still have to scroll horizontally to see the code and it's impossible to see it all without some portion of it being cut off.